### PR TITLE
Update packages, migrate to Webpack 5

### PR DESCRIPTION
**Pull request by mbmjertan**
# Description

This PR updates packages to the latest versions where it's possible to resolve dependencies (apart from WP packages), migrates Eightshift Frontend Libs to Webpack 5 and fixes issues introduced by the upgrade.

It needs heavy testing! Storybook currently doesn't work, and there might be other things that are broken as well.

### CODE REVIEW

**Comment by dingo-d** [comment]
Just a note to whoever will keep working on this: once you are ready for this to be merged to the develop branch, do an interactive rebase and squash and fix some of the commits. 
There are lots of meaningless commits that are repeated or don't have explanations. It's easier to do a changelog if the commits are clean, plus it shows others how an OS project should be maintained.

File: package.json [hunk 1]
```diff
@@ -29,7 +29,7 @@
 		"lint": "npm run lintJs && npm run lintStyle",
 		"storybookBuild": "build-storybook -c .storybook -o docs/storybook",
 		"storybook": "start-storybook -c .storybook",
-		"sassdocBuild": "node ./node_modules/sassdoc/bin/sassdoc styles/scss --dest docs/sassdocs",
+		"sassdocBuild": "node ./node_modules/sassdoc/bin/sassdoc styles/scss --dest docs/sassdocs && cp ./styles/sassdocs/main.css ./docs/sassdocs/assets/css/",
```
Comment (dingo-d) [inline package.json:32]: This could be an issue on Windows, because of path specification. We should test this.

**Review by volfkarlo — DISMISSED** [review]
One question, everything seems good 👏 
And thanks for `packages.md`

File: blocks/init/src/Blocks/custom/column/column-hooks.js [hunk 2]
```diff
@@ -33,7 +34,7 @@ const parentComponentBlock = createHigherOrderComponent((BlockListBlock) => {
 				blockClass,
 				customBlocksName,
 			]);
-			updatedProps = assign(
+			updatedProps = _.assign(
```
[the remark this answers is not in the document]
Comment (iruzevic) [reply blocks/init/src/Blocks/custom/column/column-hooks.js:37]: eslinter doesn't like due to named exports being used not sure why

**Review by mbmjertan — COMMENTED** [review]
Looks good to me (but I can't approve, as I'm the author :D)

Please check the failing Jest tests though.

**Review by gabriel-glo — DISMISSED** [review]

**Pull request merged**